HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -1151,7 +1151,7 @@ def self.apply_hvac(runner, hpxml, weather, convert_shared_systems)
         # Calculate heating capacity retention at lowest outdoor drybulb
         detailed_performance_data = heat_pump.heating_detailed_performance_data
         min_odb = detailed_performance_data.map { |dp| dp.outdoor_temperature }.min
-        next if min_odb == 47 # No temperatures below the rating temperature; don't bother to calculate a retention factor
+        min_odb = detailed_performance_data.map { |dp| dp.outdoor_temperature }.max if min_odb == 47 # No temperatures below the rating temperature; 
```

Comment:
This will result in a retention fraction >1.0 at higher temperature, is it fine?

Reply:
Rather than calculate the retention factor at the lowest ODB (which was done out of convenience), I will update the code to always calculate the factor at 5F.